### DISCUSSION TITLE
Supervisor: configuration hard without seeing docs

### DISCUSSION THREAD

**Discussion by colinfrei** [home-assistant/frontend#11267] [Other feature requests]
## The request

The new tabbed interface for addons is really nice, but unfortunately it's hard to configure an addon without seeing the documentation. At the moment that means switching between tabs, which causes all changes to be lost, or having the documentation tab open in a separate window.

My request is to display the configuration docs somewhere on the 'Configuration' tab of an addon to have a quick reference. Possibly collapsed/collapsable.

**Comment by zsarnett** [comment]
We should just add help icons for each fields.

**Comment by ludeeus** [comment]
There are no fields yet?
![image](https://user-images.githubusercontent.com/15093472/94109719-abfc7600-fe41-11ea-8429-d7e7d4b8f150.png)
